docs/src/manifolds/embedded.md

```diff
@@ -0,0 +1,43 @@
+# Embedded manifold
```

Comment:
Well, as we noticed a while back `decorated_manifold` might be misleading, since technically the embedding is decorated, but semantically `base_manifold` (i.e. the one we semantically decorate) is the manifold itself. I haven't found a good way to explain this.

Reply:
This is definitely a problem because people who would like to use embedded manifold functionality will definitely be confused without a clear explanation. At least we could write in this file that for embedded manifolds `get_embedding` returns the space the manifold is embedded in while both `base_manifold` and `decorated_manifold` return the embedded manifold.

Reply:
Yes. we can mention that.